embedfs/embed_test.go

```diff
@@ -8,17 +8,11 @@ import (
 	"testing"
 
 	"github.com/go-git/go-billy/v6"
+	"github.com/go-git/go-billy/v6/embedfs/internal/testdata"
 	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
-//go:embed testdata/empty.txt
-var singleFile embed.FS
-
-//go:embed testdata
-var testdataDir embed.FS
-
-var empty embed.FS
```

Comment:
We need this added again for `TestRead` or as a func in `provider.go`. Same for `singleFile`.